annotations/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
When building `annotations` with `mvn package`, `target` shows up as an untracked Git directory.

Reply:
Please delete this file, as the real file is in the parent directory (j2objc/.gitignore).